CHANGES.rst

```diff
@@ -20,6 +20,7 @@ New features and enhancements
 Breaking changes
 ^^^^^^^^^^^^^^^^
 * `bump2version` has been replaced with `bump-my-version` to bump the version number using configurations set in the `pyproject.toml` file. (:issue:`1557`, :pull:`1569`).
+* Xclim's units registry and units formatting are now extended from `cf_xarray`. The exponent sign "^" is now never added in the ``units`` attribute. For example, square meters are given as "m2" instead of "m^2" by xclim, both are still accepted as input (:issue:`1010`, :pull:`1590`).
```

Comment:
```suggestion
* `xclim`'s units registry and units formatting are now extended from `cf-xarray`. The exponent sign "^" is now never added in the ``units`` attribute. For example, square meters are given as "m2" instead of "m^2" by xclim, both are still accepted as input. (:issue:`1010`, :pull:`1590`).
```